### DISCUSSION TITLE
如何让“添加日期”从显示“年/月/日 时：分：秒”，变成只显示“年/月/日”

### DISCUSSION THREAD

**Discussion by MontelukastNa** [zotero-chinese/wiki#239] [求助 / Q&A] [answered]
各位彭友，请问在Zotero7/6中，如何改变主面板的“添加日期”一栏的格式？
我想让它其从显示“年/月/日 时：分：秒”，变成只显示“年/月/日”
（缩小列宽的方式不算😥）

**Comment by zepinglee** [comment]
官方目前好像没办法。

应该可以通过插件解决，不过我建议使用 yyyy-mm-dd，毕竟是 GB/T 7408 和 ISO 8601 格式。

**Reply by MontelukastNa** [reply]
> 官方目前好像没办法。
> 
> 应该可以通过插件解决，不过我建议使用 yyyy-mm-dd，毕竟是 GB/T 7408 和 ISO 8601 格式。

您好，yyyy-mm-dd也行，主要没看到类似的插件。想把hh:mm:ss去掉，不然显得好长，当然主要还是自己的阅读习惯对这“时-分-秒”部分信息不敏感。

**Reply by MontelukastNa** [reply, marked as answer]
来补充一下，Zotero-style的更新发现有针对添加日期的格式修改了，巨蟹MuiseDestiny！

**Comment by litiemei123** [comment]
请问如何变成年/月/日 时：分：秒呢，我的format是YYYY/M/D h:mm:ss，但是其实只有年月日是正确的，他的时分秒是错误的
